frontend/public/models/yaml-templates.ts

```diff
@@ -1243,6 +1244,60 @@ metadata:
 spec:
   hrefTemplate: 'https://example.com/logs?resourceName=\${resourceName}&containerName=\${containerName}&resourceNamespace=\${resourceNamespace}&podLabels=\${podLabels}'
   text: Example Logs
+`,
+  )
+  .setIn(
+    [referenceForModel(appModels.QuickStartModel), 'default'],
+    `
+apiVersion: console.openshift.io/v1
+kind: ConsoleQuickStart
+metadata:
+  name: example
+spec:
+  displayName: Deploy image
+  tags:
+  - example
+  - image
+  - deployment
+  durationMinutes: 5
+  description: Deploy an example application from an image.
+  introduction: |-
+    ### This is an example Quick Start, with which you will deploy an existing
+    image to OpenShift. For more information check QuickStart
+    [guidelines](http://openshift.github.io/openshift-origin-design/conventions/documentation/quick-starts.html).
+  tasks:
+    - title: Deploy an existing image from an image registry
+      description: |-
+        ### To deploy the **hello-openshift** image, follow these steps:
+        1. Enter the developer perspective: In the main navigation menu, click the dropdown menu and select **Developer**.
+        2. In the main navigation menu, click **Add.**
+        3. Using the project dropdown menu, select the project you would like to deploy the image in.
+        4. Click the **Container Image** tile.
+        5. In the **Image name from external registry** field, enter **openshift/hello-openshift**.
+        6. Fill in the remaining image deployment details, and then click **Create.**
+        The **Topology** view will load with your **hello-openshift-app** application. The outer rim of the larger circle represents your application, and the small circle represents the deployment.
+      review:
+        instructions: |-
+          #### Verify the image was successfully deployed:
+          Do you see a **hello-openshift** deployment?
+        failedTaskHelp: This task isn’t verified yet. Try the task again.
+      summary:
+        success: Great work! You deployed an example application using the **openshift/hello-openshift** image.
+        failed: Try the steps again.
+    - title: Run the deployed application
+      description: |-
+        ### To run the **hello-openshift** deployment application, follow these steps:
+        1. In the **Topology** view, click the deployment's white circle.
+        2. In the **hello-openshift** deployment circle, click the **Open URL** icon, which represents the route. This opens the URL and runs the application.
+      review:
+        instructions: |-
+          #### Verify your application is running:
+          In the new tab, do you see **Hello OpenShift!** ?
+        failedTaskHelp: This task isn’t verified yet. Try the task again.
+      summary:
+        success: You have deployed **hello-openshift** image.
```

Comment:
```suggestion
        success: Great work! You deployed the **hello-openshift** image.
``` 
We won't be using any success messages moving forward. So if possible, this can be removed (but if you can't remove it, I went ahead and edited it to match our other success message).